$NetBSD: patch-Cargo.toml,v 1.3 2024/05/30 06:55:09 pin Exp $

Don't require latest version of Rust to fix a bug that only affects Windows.

--- Cargo.toml.orig	2024-05-29 12:41:13.741194464 +0000
+++ Cargo.toml
@@ -10,7 +10,7 @@ homepage = "https://www.nushell.sh"
 license = "MIT"
 name = "nu"
 repository = "https://github.com/nushell/nushell"
-rust-version = "1.77.2"
+rust-version = "1.76.0"
 version = "0.94.0"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
@@ -305,4 +305,4 @@ bench = false
 # Run individual benchmarks like `cargo bench -- <regex>` e.g. `cargo bench -- parse`
 [[bench]]
 name = "benchmarks"
-harness = false
\ No newline at end of file
+harness = false
